janusgraph-core/src/main/java/org/janusgraph/graphdb/configuration/GraphDatabaseConfiguration.java

```diff
@@ -318,10 +316,15 @@ public boolean apply(@Nullable String s) {
             put("ignore-prop", IgnorePropertySchemaMaker.INSTANCE);
             put("default", JanusGraphDefaultSchemaMaker.INSTANCE);
             put("tp3", Tp3DefaultSchemaMaker.INSTANCE);
-            put("logging", LoggingSchemaMaker.DEFAULT_INSTANCE);
         }}
     );
 
+    public static final ConfigOption<Boolean> SCHEMA_MAKER_LOGGING = new ConfigOption<>(SCHEMA_NS, "logging",
+            "Controls whether logging is enabled for schema makers. This only takes effect if you set 'schema.default' " +
+            "to 'default' or 'ignore-prop'. For 'default' schema maker, warning messages will be logged before schema types " +
```

Comment:
(nitpick) I think the  "'" should be backticks to properly render this as inline code in markdown.